support/machine_test.go

```diff
@@ -0,0 +1,32 @@
+package support
+
+import (
+	"testing"
+
+	"github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	machinev1beta1 "github.com/openshift/api/machine/v1beta1"
+)
+
+func TestGetMachineSets(t *testing.T) {
+	test := NewTest(t)
+
+	machine := &machinev1beta1.MachineSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test-machineset-1",
+			Namespace: "openshift-machine-api",
+		},
+	}
+
+	test.client.Machine().MachineV1beta1().MachineSets("openshift-machine-api").Create(test.ctx, machine, metav1.CreateOptions{})
+
+	machines, _ := GetMachineSets(test)
+
+	test.Expect(machines[0].Name).To(gomega.Equal("test-machineset-1"))
```

Comment:
Can you add assertion making sure there is just one element in `machines` slice?

Reply:
Assertion Added